core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChild.java

```diff
@@ -48,4 +51,25 @@ private Integer parseIbisDocAnnotation(IbisDoc ibisDoc) {
 		}
 		return result;
 	}
+
+	/**
+	 * Calculate property overriddenFrom. Assumes that overriddenFrom has been
+	 * set already for all ancestors in the FrankElement inheritance hierarchy.
+	 */
+	public void calculateOverriddenFrom() {
+		ConfigChildKey key = new ConfigChildKey(this);
+		FrankElement match = configParent;
+		while(match.getParent() != null) {
+			match = match.getParent();
+			ConfigChild matchingChild = match.find(key);
+			if(matchingChild != null) {
+				if(matchingChild.overriddenFrom != null) {
+					this.overriddenFrom = matchingChild.overriddenFrom;
```

Comment:
Hier vind je de eerste definitie van het attribuut / het configChild. Is dat wat je wilt? Wil je niet de laatste hebben? Als je bijvoorbeeld documentatie wil overnemen, wil je die van de laatste.

Reply:
In DocWriterNew itereer ik al vanaf een child over al zijn ancestors. Dan is het niet meer nodig om op te slaan waarvandaan een method overridden is. Als je in een ancestor de method vind, dan heb je de originele method. Zal ik dit veld vervangen door een boolean?

Reply:
Ik neem bovenstaande opmerking terug. We willen geen informatie weggooien, dus de FrankElement referentie vervangen door een Boolean lijkt met toch geen goed idee. Ik heb de code veranderd zodat we in overriddenFrom de eerste match stoppen, zonder meteen de eerste definitie in de inheritance hierarchie op te zoeken.